gnovm/pkg/gnolang/values_conversions.go

```diff
@@ -175,22 +203,32 @@ GNO_CASE:
 			tv.T = t
 			tv.SetInt64(x)
 		case UintKind:
+			validate(Int8Kind, UintKind, func() bool { return tv.GetInt8() >= 0 })
+
 			x := uint(tv.GetInt8())
 			tv.T = t
 			tv.SetUint(x)
 		case Uint8Kind:
+			validate(Int8Kind, Uint8Kind, func() bool { return tv.GetInt8() >= 0 })
```

Comment:
When an int type value exceeding 255 comes in, it will exceed the range of uint8. I think overflow error should take precedence over convertable error - how do you think?

Reply:
copy paste error